### DISCUSSION TITLE
Contributors

### DISCUSSION THREAD

**Discussion by scottstirling** [scottstirling/bitcloutDAO#11] [Ideas]
Today after reviewing 24 GitHub projects with “bitclout” in the name, I starred a few of those to track and invited 3 of the leads of those to collaborate here.

Bitclout UI has updated functionality (reclouting) today which means new code has been injected or released behind Cloudflare again, from somewhere else, because currently that is the only way to update the Angular code at origin.

There were about 120,000 lines of code in main.js the other day (that’s after formatting, so, actual lines of code varies by arbitrary/conventional spec of where the formatter inserts line breaks)  I have yet to go through most of it.  But it’s pretty hard if not impossible to have 120k of human readable source code that doesn’t have at least 100 bugs and reveal information about the designer/creator, even if comments have been stripped.

**Comment by ianhernandez** [comment]
I love the reasoning here.
